database/sql/V072__total_fee_token.sql

```diff
@@ -0,0 +1,24 @@
+ALTER TABLE order_execution
+RENAME COLUMN surplus_fee TO total_fee;
+
+-- Add a new column to the order_execution table to store the total fee token
+ALTER TABLE order_execution
+ADD COLUMN total_fee_token bytea NOT NULL DEFAULT '\x0000000000000000000000000000000000000000';
+
+-- Now populate existing rows with the sell token taken from the 'orders' table, or if it doesn't exist, try from 'jit_orders'.
+UPDATE order_execution oe
```

Comment:
Did you test that this query does not leave any `surplus_token` uninitialized?

Reply:
Here are test results on mainnet prod db:

Number of rows in `order_execution`: 1466450
Number of rows being updated: 1460019

So, there are ~6k entries in `order_execution` that can't be found in `orders` or `jit_orders` (but all missing orders are older than 2 months).
My assumptions is that those are some jit orders from before we started saving them in the library. Example:
https://api.cow.fi/mainnet/api/v1/solver_competition/by_tx_hash/0x59ddfcab4910ad48b3550658f0c46c8892f114e6791dbc032c05a42c1e858b63

Note how the order `https://explorer.cow.fi/search/0x0088e45fff8487b9d4c5edcf2ad6580cdfc3a689c3c3022d7282aebf6446a8a99bd702e05b9c97e4a4a3e47df1e0fe7a0c26d2f166ff4487` can't be found in the explorer.
So, for these, I am not too worried if their token is not populated since those can't be found in explorer anyway.